README.md

```diff
@@ -117,7 +117,7 @@ packages:
       version: [">=1.0.0", "<2.0.0"]
 
     - package: calogica/dbt_expectations
-      version: [">=0.8.0", "<0.9.0"]
+      version: [">=0.9.0", "<0.10.0"]
```

Comment:
could you update this in the transform and ad_reporting README dependency matrices as well? No release needed

Reply:
good callout! updated